packages/react-components/src/components/ParticipantItem.tsx

```diff
@@ -306,9 +305,7 @@ const participantStateStringTrampoline = (
 ): string | undefined => {
   /* @conditional-compile-remove(one-to-n-calling) */
   /* @conditional-compile-remove(PSTN-calls) */
-  return props.participantState === 'Idle' || props.participantState === 'Connecting'
-    ? strings?.participantStateConnecting
```

Comment:
Is `participantStateConnecting` used anywhere else or can we remove from the strings.json files?

Reply:
I removed it, lets see if something breaks in CI